google/compute_instance.go

```diff
@@ -0,0 +1,764 @@
+// ----------------------------------------------------------------------------
```

Comment:
Ideal to remove this header since it is not, in practice, autogenerated.

Reply:
This part confused me - I thought we'd end up landing these manual resources upstream in magic-modules not by writing the files directly here?

Reply:
Either approach is acceptable!  We used the "mm-only" model for our other terraform stuff, but we don't have to.  :)

Reply:
I'd think MM would be more consistent and it *looks* like @nstogner actually generated this via MM. Probably the right thing is to start a PR on the MM repo instead?

Reply:
I'm open to either approach.  If you add it to MM, go ahead and put the file itself in `third_party`, and add it to `provider/terraform_object_library.rb`'s `copy_common_files` list.  Parallel to, e.g. `cai.go`.

Reply:
Ok, so if we make the PR to MM, the process is pretty much the same: generate the resource if the templates change, manually patch it, and add to `third_party/` ?

Reply:
Thanks, I will take that approach.

Reply:
:+1: - closed this one.